src/global.scss

```diff
@@ -45,3 +45,11 @@
 @import "./theme/sass-partials/toast";
 @import "./theme/sass-partials/typography";
 @import "./theme/sass-partials/validation";
+
+.no-results {
+  margin-top: 24px;
+  p.error-text {
```

Comment:
should prob rename this to no-results-text

Reply:
will do